fcrepo-api-x-loader/src/main/java/org/fcrepo/apix/loader/impl/LoaderRoutes.java

```diff
@@ -125,7 +125,7 @@ public void configure() throws Exception {
                 .setHeader(Exchange.HTTP_URI, header(HEADER_SERVICE_URI))
                 .setHeader("Accept", constant("text/turtle"))
                 .process(e -> LOG.info("Execution OPTIONS to service URI {}", e.getIn().getHeader(Exchange.HTTP_URI)))
-                .to("jetty:http://localhost")
+                .to("http://localhost?httpClient=#httpClient")
```

Comment:
Yeah, the param name was the thing [we weren't sure whether it was legit](https://issues.apache.org/jira/browse/CAMEL-11270), but `#` works all over.